src/demo/demo.controller.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { DemoController } from './demo.controller';
+import { DemoService } from './demo.service';
+import { Demo, PostDemoDto } from './model/demo.model';
+
+describe('DemoController', () => {
```

Comment:
Add more tests for this controller

Reply:
this is a default test file created from nest generator and is not being used in favor of e2e tests under the test/ folder